pkg/operator/ceohelpers/bootstrap.go

```diff
@@ -196,5 +196,18 @@ func IsBootstrapComplete(configMapClient corev1listers.ConfigMapLister, staticPo
 			return false, nil
 		}
 	}
+
+	// check if etcd-bootstrap member is still present within the etcd cluster membership
+	membersList, err := etcdClient.MemberList(context.Background())
+	if err != nil {
+		return false, fmt.Errorf("IsBootstrapComplete couldn't list the etcd cluster members: %w", err)
+	}
+	for _, m := range membersList {
+		if m.Name == "etcd-bootstrap" {
+			klog.V(4).Infof("(etcd-bootstrap) member is still present in the etcd cluster membership")
+			return false, nil
+		}
+	}
```

Comment:
```suggestion
	for _, m := range membersList {
		if m.Name == "etcd-bootstrap" {
			klog.V(4).Infof("(etcd-bootstrap) member is still present in the etcd cluster membership")
			return false, nil
		}
	}
```

Thanks @eranco74 for your hint 👍🏽  

So we have added this condition to `IsBootstrapComplete`, to avoid reconciling `clustermemberremovalcontroller` whenever there is `etcd-bootstrap`. 

I assume we can skip this check for SNO strategy, and this should do, or ? 🤔